utilsForFrontend.mjs

```diff
@@ -160,8 +160,8 @@ export function *inlineTexts(element, shouldTraverse = element => true) {
     for (let child of walk(element,
                            element => !(isBlock(element) ||
                                         element.tagName === 'SCRIPT' &&
-                                        element.tagName === 'STYLE')
-                                      && shouldTraverse(element))) {
+                                        element.tagName === 'STYLE') &&
```

Comment:
I think this actually make it harder to understand. I don't remember writing this (though I probably did), but here's how I read the original:

The ! lines up with the &&, suggesting that 2 clauses are ANDed together. That's what's actually going on.

When you move the 2nd &&, the 2 &&s line up exactly, and it starts looking like shouldTraverse is just one more clause in the earlier &&, unless you take the time to painstakingly balance the parens.

So I think the original indentation visually conveys the structure more clearly. Does that make sense?

Reply:
I see what you mean, though that prevents us from having an eslint rule to cover it, as leaving this as-is is not consistent (from eslint's perspective) with changing [the other one](https://github.com/mozilla/fathom/pull/116#discussion_r312537506).

I'm happy to change it back as you prefer, however.